Solutions/Digital Shadows/Playbooks/DigitalShadowsPlaybookToUpdateSeverity.json

```diff
@@ -0,0 +1,406 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "title":  "",
+        "description":  "",
+        "prerequisites":  "",
+        "prerequisitesDeployTemplateFile":  "",
+        "lastUpdateTime":  "",
+        "entities":  [
+        ],
+        "tags":  [
+        ],
+        "support":  {
+            "tier":  "community",
+            "armtemplate":  "Generated from https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Playbook-ARM-Template-Generator"
+        },
+        "author":  {
+            "name":  ""
+        }
+    },
+    "parameters":  {
+        "PlaybookName":  {
+            "defaultValue":  "Digital Shadows Playbook to Update Incident Severity",
+            "type":  "string"
+        }
+    },
+    "variables":  {
+        "azuresentinelConnectionName":  "[concat('azuresentinel-', parameters('PlaybookName'))]"
+    },
+    "resources":  [
+        {
+            "properties":  {
+                "provisioningState":  "Succeeded",
```

Comment:
Is `provisioningState` valid for a deployment template? I would've guessed that this field would be set once the template had been provisioned.

Reply:
provisioningState is set to "Succeeded" value for all other Playbooks in the Azure repo, so keeping the value unchanged.

Reply:
Sure, I can see those too. But I am looking at this one and it doesn't have it: https://github.com/Azure/Azure-Sentinel/blob/master/Playbooks/CiscoASA/CiscoASA-AddIPtoNetworkObjectGroup/azuredeploy.json

It has a similar workflowdefinition block here without it:

```
            "properties": {
                "state": "Enabled",
                "definition": {
                    "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
                    "contentVersion": "1.0.0.0",
```

I don't think we need it and should ideally remove it.